### PR TITLE
one blocked messages can be a fraction of a pct

### PR DESCRIPTION
**Pull request by zippy1981**

### CODE REVIEW

**Review by bran-solo — CHANGES_REQUESTED** [review]
Whoops didn't see you were waiting for my approval.

I don't see a change for the actual config here. Can we please drop the rate for everyone not-kero to 0.5%?

**Comment by zippy1981** [comment]
Config is not stored in git. It's overridden in .env. I just keep defaults
in config.js

**Review by bran-solo — APPROVED** [review]
Approved. How do we make the .env change, is this now some wacky ssh thing?

**Pull request merged**
